srtp/srtp.c

```diff
@@ -74,39 +74,50 @@ srtp_debug_module_t mod_srtp = {
 };
 
 #define octets_in_rtp_header 12
-#define uint32s_in_rtp_header 3
 #define octets_in_rtcp_header 8
-#define uint32s_in_rtcp_header 2
-#define octets_in_rtp_extn_hdr 4
+#define octets_in_rtp_xtn_hdr 4
+
+static uint32_t srtp_get_rtp_hdr_len(const srtp_hdr_t *hdr)
```

Comment:
I don't think it's semantically right to use `uint32_t` for byte lengths?  I'd suggest `size_t`.  (Not that this matters practically.)

Reply:
I totally agree and plan to convert to size_t both in the public API and internally, at the moment the mix of size_t, uint32_t, int and unsigned int is just staggering in this project. But such a change in the public API needs a major version bump which is why I have been putting it off. I think it might be a goal for the Christmass break to start.